packages/components/src/utils/sendToast.tsx

```diff
@@ -0,0 +1,53 @@
+import type { NotificationType, NotificationProps } from 'hds-react';
+import { Notification } from 'hds-react';
+import type { ToastContentProps } from 'react-toastify';
+import { toast } from 'react-toastify';
+import useCommonTranslation from '../hooks/useCommonTranslation';
+
+type NotificationOptions = Omit<Partial<NotificationProps>, 'size'>;
+
+type SendToastArgs = {
+  title: string;
+  message: string;
+  type: NotificationType;
+  options?: NotificationOptions;
+};
+
+type ToastProps = {
+  closeToast?: ToastContentProps['closeToast'];
+} & SendToastArgs;
+
+function Toast({ closeToast, title, message, type, options }: ToastProps) {
+  const { t } = useCommonTranslation();
+
+  return (
+    <Notification
+      label={title}
+      position="top-right"
+      dismissible
+      closeButtonLabelText={t('common:button.close')}
+      type={type}
+      onClose={closeToast}
+      {...options}
+    >
+      {message}
+    </Notification>
+  );
+}
+
+export default function sendToast({
+  title,
+  message,
+  type,
+  options,
+}: SendToastArgs) {
+  toast((content: ToastContentProps<unknown>) => (
+    <Toast
+      {...content}
+      title={title}
+      message={message}
+      type={type}
+      options={options}
+    />
+  ));
+}
```

Comment:
We already have a `ToastContainer` -component and we are sending the toasts `toast.error(t('search:errorLoadMore'));`. Does this differ somehow?

Reply:
it would be good if @nenonja, @melniiv or @kristianjokela would have time to review this further.

Reply:
> We already have a `ToastContainer` -component and we are sending the toasts `toast.error(t('search:errorLoadMore'));`. Does this differ somehow?

I tried using the toast.error from react-toastify and calling it in GeolocationProvider.tsx's try clause like:
```javascript
...
import { toast } from 'react-toastify';
...
try {
      geolocation = await geolocationService.getCurrentPosition();
      toast.error(t('error.description.permission_denied'));
      setLocation(geolocation);
    }
```
but the result was an error message with a big (i) at the bottom of the screen—here's a bottom part of a screenshot showing what it looks like:
![toast_error](https://user-images.githubusercontent.com/77663720/212648655-93b08b73-8d32-4f2a-ba8a-f9233ff41d40.png)

So if the toast.error doesn't work anywhere else either then a bug ticket might be good to be made out of that but to get this merged I'd go with the showToast function now as it is currently working ok according to testing.